blog/2020-12-16-cwa-release-1-9/index.md

```diff
@@ -18,7 +18,7 @@ If a user had multiple encounters with a high risk (red), he or she will see the
 
 <br></br>
 
-<center> <img src="./cwa-high-and-low-risk.jpg" title="Corona-Warn-App High and Low Risk" alt="Corona-Warn-App High and Low Risk" style="align: center"> </center>
+<div style="text-align:center"> <img src="./cwa-high-and-low-risk.jpg" title="Corona-Warn-App High and Low Risk" alt="Corona-Warn-App High and Low Risk" style="align: center"> </div>

```

Comment:
`style="align: center"` causes
"Error: CSS: align: Property align doesn't exist."

Reply:
As of the most recent version of this PR, this is fixed. Thanks for the information